Userland/Libraries/LibGfx/Path.h

```diff
@@ -176,6 +176,8 @@ class Path {
         return m_split_lines.value();
     }
 
+    void clear() { m_split_lines.clear(); }
+
```

Comment:
This is just a cache though, the stuff in the path aren't being cleared, so the name might be misleading

Reply:
Ahh yeah, I wanted to clear the segments instead. I'll rework this